### ISSUE TITLE
Prepare 2021 Sponsor Prospectus

### ISSUE THREAD

**Issue by altsalt** [SeaGL/organization#110]

**Comment by altsalt** [comment]
Blocked by #57 and #75

**Comment by lufthans** [comment]
Two proposed prospecti. One is with Media sponsor in the table with the other sponsorships, the other with Media sponsor at the end.

Still open to ideas on what to offer sponsors and sponsorship requests to make.

https://docs.google.com/document/d/14lR5GuMzsFO6BR7_5uwm7u44xeFUYOmxNie9UOp01E8/edit#

https://docs.google.com/document/d/1AkHY1HcA2k222rJHwsZRE6I_2LuBxlXxJEmnwIEFLis/edit?pli=1#heading=h.3yeirzwne22u

**Comment by lufthans** [comment]
# SeaGL 2021 Sponsorship Levels

The below is proposal / questionaire. It's a first draft at documentation for in
ternal use and subject to accepting better ideas :).

Please let us know if you have feedback.

## Sponsorship Levels

Sponsorship levels ( Metal, Local Business, Non-exhibiting, Educational / Nonprofit and Media ) are documented in the sponsorship prospectus.

https://docs.google.com/document/d/14lR5GuMzsFO6BR7_5uwm7u44xeFUYOmxNie9UOp01E8/edit#

## Sponsorship Add-ons

In addition to the general levels, we have alternative sponsorships / add-ons tahat can be paid for.

Those have only short names in the prospectus. This longer internal descriptions is for our reference when answering sponsor questions.

We don't want the sponsor prospectus to become noisy or turn into a novella, but we can add clarification as we determine it will help.

This document will also describe how the add-ons will hypothetically work for a virtual conference in 2021.

### Track (per track, per day):  $500

The program committee will determine how many tracks we will have ( including if there will be more than one ) once they've got an idea of which talks will be accepted.

### Hallway Track ( per day ):  $500

For a virtual conference, the hallway track is our main IRC channel/lobby. Sponsor would get a mention in the channel topic and during the conference.

### TeaGL Virtual Afternoon Tea Social Event:  $250

TeaGL was a great experience last year. It was also popular. It is a planned social event. For the virtual conference it runs both Friday and Saturday afternoons during the conference.

We also had a TeaGL tea exchange that ran before the conference, which built familiarity and interest in TeaGL.

For the virtual conference TeaGL participants are encouraged to make a mug of tea or other beverage and join us in the video chat for a relaxed social gathering.

### Coffee Vouchers ( both days ):  $1,000

For a virtual conference we can make coffee shop vouchers available to attendees, so they can get a local coffee to enjoy during the morning coffee breaks on both days of the conference.

The sponsorship would also include the virtual coffee break social event.

### Friday / Saturday Attendee Lunch Vouchers:  $1,000 per day

For a virtual conference we can make restaurant vouchers available to attendees.

The vouchers could be for one or more delivery services or directly for restaurants.

### Career Expo Platinum:  $2,000

In 2020 we added a Career Expo enhancing our existing job board with career counselors providing career advice and resume reviews.

The sponsorship would include mentions in Career Expo promotion and during the event.

For the top level sponsorship it would also include social media posts and mention in the Career Expo IRC channel topics.

### Career Expo Gold:  $1,000

In 2020 we added a Career Expo enhancing our existing job board with career counselors providing career advice and resume reviews.

The sponsorship would include mentions in Career Expo promotion and during the event.

### Digital Sticker Pack:  $500

The digital stickers are for applications such as Telegram. Sponsorship would get mention on the download page and the sponsor logo as one of the stickers.

### IDEA: Inclusion, Diversity, Equity, Access :  $1,000

This sponsorship funds our IDEA initiative.

It would fund the IDEA effort and committee ( travel, etc ) for IDEA outreach as well.

### Live Transcriptions (per track):  $3,000

As part of our accessibility effort, we would love to feature live transcription of talks. The sponsorship would fund the live transcription and be recognized during talk announcements and in the room IRC topic.

### Saturday Post-Event Party:  $2,000

Sponsors would be recognized during the post-event party on Saturday night.

### Volunteer Appreciation Event:  $500

Salt gets your logo sculpted into his hair.

What do we want to do for a virtual volunteer appreciation event on Sunday?

Sponsored naps? :)

We could provide vouchers for meal delivery or massages.


## General Questions

* Do we want to consider something other than metals ( Platinum, Gold, etc ) as the main sponsorship levels?

* Can we add a text mention in Matrix? Can we include sponsor logos?

* Can we add a text mention in jitsi or BBB? Can we include sponsor logos?

* Do we want to include sponsor info for ad hoc rooms?

* For Career Expo, if we have multiple levels of sponsorship, should we choose something other than the metal levels that are used for the main sponsoships?

* Should we drop post-event party and volunteer event sponsorships?

* Do we want to offer discounts for next year?
   * e.g. sponsor at same or higher level as last year and get 10% off this year?

* We have two founding sponsors and a couple that are close to founding sponsors
   * Would we like to create a long-term sponsorship recognition?

## Traditional Add-ons options dropped due to virtual only conference

* Lanyards

* Attendee Swag Bag

**Comment by funnelfiasco** [comment]
My $0.02 (whatever sponsorship level that is)

>  Do we want to consider something other than metals ( Platinum, Gold, etc ) as the main sponsorship levels?

Maybe for next year. Metals are well-known in this space, so doing something different would require a lot of consideration that we probably don't want to prioritize at this point.

> Can we add a text mention in Matrix? Can we include sponsor logos?

That might be hard to do unless we make it someone's job to post the mentions regularly. I don't think we'd want to take up the topic with a list of sponsors.

>  Can we add a text mention in jitsi or BBB? Can we include sponsor logos?

Putting logos on the "ad roll" would be good. I'm not sure we'd want to bother with anything else. Not opposed to it, though.

>  Do we want to include sponsor info for ad hoc rooms?

Sounds too complicated unless there's someone who very specifically wants to give us money for only that.

> For Career Expo, if we have multiple levels of sponsorship, should we choose something other than the metal levels that are used for the main sponsoships?

It might be good so that they don't overlap. I'm not sure we need multiple levels for this, but I defer to your experience with it.

>  Should we drop post-event party and volunteer event sponsorships?

Yes, unless we use the sponsorship to pay for a performer to ...perform... at the party. We may also want to consider dropping the coffee and lunch sponsorships this year. Since we don't have registration and people could be anywhere, distributing vouchers sounds like a really unpleasant task to give someone.

>  Do we want to offer discounts for next year?
>     e.g. sponsor at same or higher level as last year and get 10% off this year?

I like that!

 >  We have two founding sponsors and a couple that are close to founding sponsors
 >    Would we like to create a long-term sponsorship recognition?

It's a good idea, but maybe save it for our 10th SeaGL? Just in the interests of time.

**Comment by altsalt** [comment]
Did some market research, here are some relevant links:
- https://stackconf.eu/sponsor/
- https://www.fosscon.us/Sponsors
- https://cposc.org/sponsor/
- https://olfconference.org/sponsors/sponsorship-application/
- https://2020.allthingsopen.org/become-a-sponsor/
- https://apache.org/foundation/sponsorship.html
- https://events.linuxfoundation.org/open-source-summit-north-america/sponsor/
- https://events.linuxfoundation.org/wp-content/uploads/2021/06/sponsor-plumberscon21-060421.pdf

We need to get the prospectus up and done ASAP. I've checked it off meeting wise, @sramkrishna and @lufthans what is left to export and send LF a v1?

**Comment by lufthans** [comment]
> My $0.02 (whatever sponsorship level that is)
> 
> > Do we want to consider something other than metals ( Platinum, Gold, etc ) as the main sponsorship levels?
> 
> Maybe for next year. Metals are well-known in this space, so doing something different would require a lot of consideration that we probably don't want to prioritize at this point.

Yeah, low priority, but I think we should ask every few years.
 
> > Can we add a text mention in Matrix? Can we include sponsor logos?
> 
> That might be hard to do unless we make it someone's job to post the mentions regularly. I don't think we'd want to take up the topic with a list of sponsors.

Yeah, no need to turn the topic into a short essay.

Is there a way to have a room welcome message in Matrix?

Can we add vanity roles in Matrix? If so, we could have a sponsor role/title.

> > Can we add a text mention in jitsi or BBB? Can we include sponsor logos?
> 
> Putting logos on the "ad roll" would be good. I'm not sure we'd want to bother with anything else. Not opposed to it, though.

Yes, I'll add "add roll" to the checklist, thanks for the reminder.
 
> > Do we want to include sponsor info for ad hoc rooms?
> 
> Sounds too complicated unless there's someone who very specifically wants to give us money for only that.
> 
> > For Career Expo, if we have multiple levels of sponsorship, should we choose something other than the metal levels that are used for the main sponsoships?
> 
> It might be good so that they don't overlap. I'm not sure we need multiple levels for this, but I defer to your experience with it.

We decided to go with a single sponsorship level for Career Expo, but it can have multiple sponsors at that level.
 
> > Should we drop post-event party and volunteer event sponsorships?
> 
> Yes, unless we use the sponsorship to pay for a performer to ...perform... at the party. We may also want to consider dropping the coffee and lunch sponsorships this year. Since we don't have registration and people could be anywhere, distributing vouchers sounds like a really unpleasant task to give someone.

We decided to keep it in case it's enticing to a sponsor.

> > Do we want to offer discounts for next year?
> > e.g. sponsor at same or higher level as last year and get 10% off this year?
> 
> I like that!

I'll work on adding it to our internal documentation.

> > We have two founding sponsors and a couple that are close to founding sponsors
> > Would we like to create a long-term sponsorship recognition?
> 
> It's a good idea, but maybe save it for our 10th SeaGL? Just in the interests of time.

I was thinking of trying to roll out founding sponsors this year and add the 10% discount for next year, but it could be the other way around or even neither this year.

In any case, documentation first.

**Comment by altsalt** [comment]
> > > Can we add a text mention in Matrix? Can we include sponsor logos?
> > 
> > 
> > That might be hard to do unless we make it someone's job to post the mentions regularly. I don't think we'd want to take up the topic with a list of sponsors.
> 
> Yeah, no need to turn the topic into a short essay.
> 
> Is there a way to have a room welcome message in Matrix?
> 
> Can we add vanity roles in Matrix? If so, we could have a sponsor role/title.

Pinging @AndrewKvalheim on these since he's currently exploring Matrix in greater depth.

**Comment by funnelfiasco** [comment]
Added a link to 2021 sponsors to the sponors page now that the prospectus is live on the website.

**Comment by AndrewKvalheim** [comment]
> Can we add vanity roles in Matrix? If so, we could have a sponsor role/title.

Not yet. [Flair] would be appropriate for this but it’s still [under development][MSC3219].

> Is there a way to have a room welcome message in Matrix?

Sort of; there’s no first-class support for [welcome messages], and [message pinning] isn’t ready to use yet, but we have a couple things to work with:

- We can use [widgets] to embed arbitrary web content at the top of a room. In conference sessions this would be a live stream or video call ([screenshots](https://matrix.org/blog/2021/02/15/how-we-hosted-fosdem-2021-on-matrix)), but perhaps a central room could display some general messaging and a sponsors list.
- We’ll likely be running a simple bot for other purposes and could probably set it up to greet attendees individually.

[flair]: https://github.com/matrix-org/matrix-doc/blob/6c6aa0b3d6eb6edc65c68ad78072a6733998969b/proposals/1772-groups-as-rooms.md#flair
[MSC3219]: https://github.com/matrix-org/matrix-doc/pull/3219
[welcome messages]: https://github.com/vector-im/element-web/issues/1774
[message pinning]: https://github.com/vector-im/element-web/issues/6606
[widgets]: https://matrix.org/blog/2017/08/23/introducing-matrix-widgets

**Comment by lufthans** [comment]
Sounds like there's nothing we can promise with matrix, but we can have a wishlist request for that might turn out something that works. Thanks!

**Comment by altsalt** [comment]
Are we good to close this issue out? Any other issues that need to be created?

**Comment by koronkowy** [comment]
> Are we good to close this issue out? Any other issues that need to be created?

The prospectus is live and being sent out to folks so, it should probably be closed.

I added some issues re: prospectus visibility though and 2021 partnership stuff:
https://github.com/SeaGL/organization/issues/162
https://github.com/SeaGL/organization/issues/161
https://github.com/SeaGL/organization/issues/160

**Comment by altsalt** [comment]
Agreed at 2021-09-16 finance/partnerships meeting.

**Comment by koronkowy** [comment]
Maybe possible + info about graphical recording ask to be included in the prospectus coming?
